src/containers/support/SupportOverview.tsx

```diff
@@ -6,7 +6,7 @@ import { SupportSection } from './Support';
 
 const headerProps = {
   description:
-    "Welcome to support for the VA Lighthouse API program. You can visit our FAQ page for answers to common questions. If you need to submit a support request or report a bug, you can contact us via GitHub. For general feedback use our 'Contact Us' form. Our customer support team is happy to help and will respond within one business day.",
+    "Welcome to support for the VA Lighthouse API program. You can visit our FAQ page for answers to common questions. For general feedback use our 'Contact Us' form. Our customer support team is happy to help and will respond within one business day.",
```

Comment:
This removes the wording for support requests entirely which we're now trying to funnel through the contact us form.

```suggestion
    "Welcome to support for the VA Lighthouse API program. You can visit our FAQ page for answers to common questions. For support and/or general feedback use our 'Contact Us' form. Our customer support team is happy to help and will respond within one business day.",
```

Reply:
good catch on this content!  I agree.